Kudu.Core/Functions/KedaFunctionTriggerProvider.cs

```diff
@@ -239,25 +253,25 @@ internal static string GetKedaTriggerType(string triggerType)
 
             switch (triggerType)
             {
-                case "queuetrigger":
+                case TriggerTypes.AzureStorageQueue:
```

Comment:
I'm wondering why you change it? I'd like to know the reason.

Reply:
Modified the strings to use the constants consistently across the file, this way any changes to the constants can reflect in all the places.

Reply:
Thanks. That makes sense.